cli_tools/gce_vm_image_import/importer/processor_test.go

```diff
@@ -0,0 +1,125 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"testing"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/mocks"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"github.com/stretchr/testify/assert"
+	v1 "google.golang.org/api/compute/v1"
+)
+
+func TestDefaultProcessorProvider_InspectDataDisk(t *testing.T) {
+	processorProvider := defaultProcessorProvider{
+		ImportArguments{
+			WorkflowDir: "testdata",
+			DataDisk:    true,
+		},
+		mockComputeDiskClient{},
+		mockDiskInspector{},
+	}
+
+	processors, err := processorProvider.provide(persistentDisk{})
+	assert.NoError(t, err)
+	assert.Equal(t, 1, len(processors), "there should be 1 processor, got %v", len(processors))
+	_, ok := processors[0].(*dataDiskProcessor)
+	assert.True(t, ok, "processor is not dataDiskProcessor")
+}
+
+var uefiTests = []struct {
+	isUEFIDisk               bool
+	isInputArgUEFICompatible bool
+}{
+	{isUEFIDisk: true, isInputArgUEFICompatible: false},
+	{isUEFIDisk: false, isInputArgUEFICompatible: false},
+	{isUEFIDisk: true, isInputArgUEFICompatible: true},
+	{isUEFIDisk: false, isInputArgUEFICompatible: true},
+}
```

Comment:
Unused in this file (it's referenced in other files, so you'll want to move/copy to the files where you're referencing it.)

Reply:
done